### PR TITLE
Added site of kriskirov.

### PR DESCRIPTION
**Pull request by kriskirov**

### CODE REVIEW

File: _pages/kriskirov.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: kriskirov
+pageUrl: kriskirov.github.io
+timestamp: TODAYS-24-2017-04-Mo
+---
```
Comment (brianamarie) [inline _pages/kriskirov.yaml]: Hi @kriskirov! A few changes that may need to happen before the tests pass are:
- Make sure the date is formatted like `timestamp: 2017-04-24` 
- Try adding an extra line break after line 5

Let me know if you want or need any help 😄

Comment (kriskirov) [inline _pages/kriskirov.yaml]: Thanks :)

Comment (brianamarie) [inline _pages/kriskirov.yaml]: @kriskirov You're getting closer - it has to be year first, then month, then day of month. 👍

Comment (kriskirov) [inline _pages/kriskirov.yaml]: I can't see any notifications on the review You make. Is it by design? Is it intended?

Comment (kriskirov) [inline _pages/kriskirov.yaml]: Ah.. maybe I should use @brianamarie

Comment (kriskirov) [inline _pages/kriskirov.yaml]: @brianamarie I understand that I have provided the wrong URL for the link, but the URL provided in my Settings tab is of another repository. Is there no way to add a link to a non-repository?

Comment (brianamarie) [inline _pages/kriskirov.yaml]: @kriskirov Good question. Add the link that is show in your settings of the repository - as long as it's a valid URL and you omit the `http` portion, it should pass the test.

Comment (kriskirov) [inline _pages/kriskirov.yaml]: @brianamarie  Thanks! I've finally nailed it.

**Pull request merged**